docker-compose/is/conf/identity-server/repository/conf/deployment.toml

```diff
@@ -12,21 +12,28 @@ create_admin_account = true
 type = "database_unique_id"
 
 [database.identity_db]
-url = "jdbc:mysql://mysql:3306/WSO2IS_IDENTITY_DB?autoReconnect=true&amp;useSSL=false"
+type = "h2"
+url = "jdbc:h2:./repository/database/WSO2IDENTITY_DB;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000"
 username = "wso2carbon"
 password = "wso2carbon"
-driver = "com.mysql.cj.jdbc.Driver"
-[database.identity_db.pool_options]
-validationQuery="SELECT 1"
 
 [database.shared_db]
-url = "jdbc:mysql://mysql:3306/WSO2IS_SHARED_DB?autoReconnect=true&amp;useSSL=false"
+type = "h2"
+url = "jdbc:h2:./repository/database/WSO2SHARED_DB;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000"
 username = "wso2carbon"
 password = "wso2carbon"
-driver = "com.mysql.cj.jdbc.Driver"
-[database.shared_db.pool_options]
-validationQuery="SELECT 1"
 
 [keystore.primary]
 file_name = "wso2carbon.jks"
 password = "wso2carbon"
+
+[truststore]
+file_name="client-truststore.jks"
+password="wso2carbon"
+type="JKS"
+
+[account_recovery.endpoint.auth]
```

Comment:
Is it because we are hosting portals externally?

Reply:
It uses, If the authenticationendpoint web app is hosted externally.

Reply:
Addressed in https://github.com/wso2/docker-is/pull/366/commits/91275c755f25243bc73220b51b86c90e25515291